src/screens/SurveyOverviewScreen.tsx

```diff
@@ -0,0 +1,83 @@
+import { useNavigation } from '@react-navigation/core';
+import { StackNavigationProp } from '@react-navigation/stack';
+import React, { useCallback } from 'react';
+import { useQuery } from 'react-apollo';
+import { SectionList } from 'react-native';
+
+import { SectionHeader, TextListItem } from '../components';
+import { LoadingSpinner } from '../components/LoadingSpinner';
+import { texts } from '../config';
+import { combineLanguages } from '../helpers';
+import { usePullToRefetch, useSurveyLanguages } from '../hooks';
+import { SURVEYS } from '../queries/survey';
+import { Survey } from '../types';
+
+const useSurveySections = () => {
+  const { data: surveys, loading, refetch } = useQuery<{
+    ongoing: Survey[];
+    archived: Survey[];
+  }>(SURVEYS, { fetchPolicy: 'cache-and-network' });
+
+  const sectionData = [];
+
+  if (surveys?.ongoing.length) {
+    sectionData.push({ data: surveys?.ongoing, key: 'ongoing' });
+  }
+
+  if (surveys?.archived.length) {
+    sectionData.push({ data: surveys?.archived, key: 'archived', title: texts.survey.archive });
+  }
+
+  return { loading, refetch, surveySections: sectionData };
+};
+
+const parseSurveyToItem = (survey: Survey, languages: string[]) => {
+  const langTitle = combineLanguages(languages, survey.title);
+
+  // we know there will be at least one language for the question
+  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+  const langQuestion = combineLanguages(languages, survey.questionTitle)!;
+
+  const title = langTitle?.length ? langTitle : langQuestion;
+
+  return { title, routeName: 'SurveyDetail', params: { id: survey.id } };
+};
+
+const renderSectionHeader = ({
+  section: { title, data }
+}: {
+  section: { title?: string; data: Survey[] };
+}) => {
+  if (!data.length || !title) return null;
+
+  return <SectionHeader title={title} />;
+};
+
+export const SurveyOverviewScreen = () => {
```

Comment:
i think here is some landscape stuff missing:

<img width="1000" alt="Bildschirmfoto 2021-06-18 um 16 39 31" src="https://user-images.githubusercontent.com/1942953/122578029-c3c0fd80-d053-11eb-9e97-a09e8fac6d32.png">

Reply:
yes and no.
the safe area view is missing, but it not having a cutoff on both sides is in line with the tours and locations view

![Screenshot 2021-06-18 at 16 42 08](https://user-images.githubusercontent.com/59824597/122578463-2d410c00-d054-11eb-97ae-c638a03fd052.png)

Reply:
added safe area view now

Reply:
nice

<img width="1054" alt="Bildschirmfoto 2021-06-18 um 17 00 49" src="https://user-images.githubusercontent.com/1942953/122580902-c07b4100-d056-11eb-8f41-7d1fc1908a46.png">